### ISSUE TITLE
Impossible to use training sets kept in other location than INSTALLATION_DIR

### ISSUE THREAD

**Issue by pdec** [linsalrob/PhiSpy#31]
I noticed that the below check forces users to keep their retrained reference datasets in INSTALLATION_DIR/PhiSpyModules/data directory 
https://github.com/linsalrob/PhiSpy/blob/540fbf2548cd6580a6fd9d0a8e003226b1044710/PhiSpyModules/classification.py#L34-L36

The same will happen with kmers file within makeTest.py and makeTrain.py 
https://github.com/linsalrob/PhiSpy/blob/540fbf2548cd6580a6fd9d0a8e003226b1044710/PhiSpyModules/makeTest.py#L16-L20

Hence, while retraining one needs to point the PhiSpy's installation dir (e.g. `/usr/local/lib/python3.6/site-packages/PhiSpy...`) and can't keep separate trainSets dirs.

**Comment by linsalrob** [comment]
I agree with this issue, but will not refactor the code for the current release. We will work on this for the next release.
